apps/cowswap-frontend/src/common/constants/banners.ts

```diff
@@ -4,4 +4,5 @@ export const BANNER_IDS = {
   LIMIT_ORDERS_INFO: 'limitOrders_showInfoBanner',
   COW_AMM: 'cow_amm_banner_2024_va',
   ANNOUNCEMENT: 'announcementBannerClosed',
+  HIRING_SPEECH_BUBBLE: 'cow_hiring_speech_bubble_v1',
```

Comment:
Oh boy, these banner ids are very eclectic.
Mind sticking to camelCase, just because that's the usual localStorage key format we use?
And separate the version with `:v1`.
Or better yet, start with v0?